src/js/components/Notification/stories/typescript/Status.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import { Box, Grommet, Notification, Text } from 'grommet';
+
+import { deepMerge } from 'grommet/utils';
+import { grommet, ThemeType } from 'grommet/themes';
+import { DisabledOutline } from 'grommet-icons';
+
+// Type annotations can only be used in TypeScript files.
+// Remove ': ThemeType' if you are not using Typescript.
+const customTheme: ThemeType = deepMerge(grommet, {
+  global: {
+    font: {
+      family: 'Arial',
+    },
+  },
+  notification: {
+    container: {
+      pad: { horizontal: '200px', vertical: '100px' },
+      background: '#aaccff',
+    },
+    iconContainer: {
+      pad: '10px',
+    },
+    textContainer: {
+      gap: '25px',
+    },
+    title: {
+      color: '#0223df',
+    },
+    message: {
+      color: {
+        light: '#0223df',
+        dark: '#6101a0',
+      },
+    },
+    close: {
+      icon: DisabledOutline,
+    },
+    unknown: {
+      color: '#b0ff7b',
+    },
+  },
+});
+
+export const TSStatus = () => (
```

Comment:
Instead of this being a story about "Status", what if it were simply about "Themed"?

```suggestion
export const Themed = () => (
```